Server/src/services/DataUpload/EditUploadService.ts

```diff
@@ -48,13 +48,6 @@ export default class EditUploadService extends AbstractUploadService {
         let arrayOfDatasetInfo = [this.datasetId, this.parsedFileData.dataset_name, dataSetDataTypeID, publicationID, subcategoryID, this.parsedFileData.data.comments]
         await this.insertDataset(this.uploadModel, arrayOfDatasetInfo)
 
-        // Run check on variable vs contents length to see if they're equal for data points and insert
-        if (this.parsedFileData.data.variables.length == this.parsedFileData.data.contents[0].point.length) {
```

Comment:
Was there a reason this condition is removed?

Reply:
this condition was bugged and would throw, it should be fixable, but the case is unlikely, not a security threat, so better spend time on higher priority